src/libraries/Microsoft.VisualBasic.Core/tests/NewLateBindingTests.cs

```diff
@@ -96,6 +96,7 @@ public static IEnumerable<object[]> LateCall_OptionalValues_Data()
 
         [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsReflectionEmitSupported))]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/51834", typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsBrowser))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/51834", typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsAppleMobile))]
```

Comment:
Just a note on this, for disabling the tests I would add `nameof(PlatformDetection.IsMonoRuntime)` in the pattern, because using just:
`nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsAppleMobile))` would also disable the test for running it with NativeAOT (and the ActiveIssue "sort-of" does not really match)